batch.go

```diff
@@ -422,16 +407,34 @@ func (batch *batch) isCategory() error {
 	return nil
 }
 
-// isPaymentTypeCode checks that the Entry detail records have either:
-// "R" For a recurring WEB Entry
-// "S" For a Single-Entry WEB Entry
-func (batch *batch) isPaymentTypeCode() error {
-	for _, entry := range batch.Entries {
-		if !strings.Contains(strings.ToUpper(entry.PaymentTypeField()), "S") && !strings.Contains(strings.ToUpper(entry.PaymentTypeField()), "R") {
-			// TODO dead code because PaymentTypeField always returns S regardless of Discretionary Data value
-			msg := fmt.Sprintf(msgBatchWebPaymentType, entry.PaymentTypeField())
-			return &BatchError{BatchNumber: batch.Header.BatchNumber, FieldName: "PaymentType", Msg: msg}
-		}
+// categoryForwardAddenda02 verifies CategoryForward Addenda02 TypeCode is 02
+func (batch *batch) categoryForwardAddenda02(entry *EntryDetail, addenda Addendumer) error {
+	return nil
```

Comment:
This needs an implementation right?

Reply:
This seems like test cases should fail. Otherwise, we need some test cases that do fail!